po/de.po

```diff
@@ -277,10 +277,10 @@ msgid "Build a tracked sniffer"
 msgstr "Baut einen Kettenschnüffler"
 
 msgid "Build a vault"
-msgstr ""
+msgstr "Baut einen Speicher"
```

Comment:
Minor inconsistency: Vault is translated as *Bunker* (object name/SatCom).

Reply:
What use is the structure? Storing data? Then "Speicher" is correct. "Bunker" has the same meaning in German and English: A place to protect people in when (ballistic) attacks happen.

Reply:
In that case *Bunker* would indeed be correct, it does not store data.

Reply:
https://colobot.info/SatComOnline/dev/object/safe

Reply:
Alright. Sorry. I mistook that to be part of the signal infrastructure. Integrated and committed, Do I have to do anything to the PR to reflect the changes in that or can you pull the newest commits along?

Reply:
SatCom in the browser is seriously cool!

Reply:
Everything checks out and it's ready to be merged. Thank you!